psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -93,9 +93,9 @@ def scf_compute_energy(self):
             # die_if_not_converged()
             raise e
         else:
-            core.print_out("  Energy did not converge, but proceeding anyway.\n\n")
+            core.print_out("  Energy and wave function did not converge, but proceeding anyway.\n\n")
```

Comment:
and/or :)

Reply:
whatever you do, don't get info specifics or the geom opt message will be a mess.

what, you're siding with JCP on "wave function", not "wavefunction"!?

Reply:
Compound words in English are a nightmare :stuck_out_tongue_closed_eyes:

But yes, it looks like it might be a difference of lingo: physicists probably favor "wave function" whereas chemists favor "wavefunction". I guess I intrinsically favor the former. It also has an order of magnitude more hits on Google Scholar ;)